{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/settings/dist.py

```diff
@@ -35,32 +35,14 @@
             'class': 'logging.StreamHandler',
             'formatter': 'verbose'
         },
-        "debug_file_handler": {
-            "class": "logging.handlers.RotatingFileHandler",
-            "level": "DEBUG",
-            "formatter": "verbose",
-            "filename": os.path.join(BASE_DIR, 'logs', 'debug.log'),
-            "maxBytes": 10485760,
-            "backupCount": 20,
-            "encoding": "utf8"
-        },
-        "info_file_handler": {
-            "class": "logging.handlers.RotatingFileHandler",
-            "level": "INFO",
-            "formatter": "verbose",
-            "filename": os.path.join(BASE_DIR, 'logs', 'info.log'),
-            "maxBytes": 10485760,
-            "backupCount": 20,
-            "encoding": "utf8"
-        }
     },
     'loggers': {
         'apps': {
             "level": "DEBUG",
-            "handlers": ["debug_file_handler", "info_file_handler"],
+            "handlers": ["console"],
```

Comment:
I have one request related to logging: please use the JSON format for log messages in production: this will play nicely with the CloudWatch or any other cloud log ingestors (especially in containerized deployments). Please also pay attention to apply the JSON format to the log errors, especially the stack traces.

Reply:
@Struchu this is bigger than a quick fix, isn't? I'd rather have a separate PR for that.

Theres https://github.com/madzak/python-json-logger but I haven't researched how easy it is to integrate with Django config. It should work out of the box, BUT.

Reply:
OK, fine by me :)